server/queue/queue-api/src/test/java/org/apache/james/queue/api/DelayedMailQueueContract.java

```diff
@@ -79,7 +79,7 @@ default void enqueueWithReasonablyLongDelayShouldDelayMail(ExecutorService execu
     default void enqueueWithVeryLongDelayShouldDelayMail(ExecutorService executorService) throws Exception {
         getMailQueue().enQueue(defaultMail()
             .build(),
-            100 * 365,
+            Long.MAX_VALUE,
```

Comment:
Changing delay to `Long.MAX_VALUE` caused `java.lang.ArithmeticException: long overflow` in `MemoryMailQueue::enQueue` then CI build fails. Please take a little fixup from `trantienduchn:JAMES-2309`

Reply:
done, thx